### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Tengiz-Merabishvili**

### CODE REVIEW

File: src/CompetitionPlatform/Controllers/UserProfileController.cs [hunk 1]
```diff
@@ -280,5 +293,16 @@ public static bool IsGuid(string value)
             Guid x;
             return Guid.TryParse(value, out x);
         }
+
+        private async Task<bool> UserIsLykkeMember(string email)
+        {
+            var userRole = await _userRolesRepository.GetAsync(email);
+            var isAdmin = (userRole != null) && userRole.Role == StreamsRoles.Admin;
```
Comment (justinlevinson) [inline src/CompetitionPlatform/Controllers/UserProfileController.cs]: Do we have admins that aren't registered with a lykke.com or lykkex.com address?

File: src/CompetitionPlatform/Controllers/UserProfileController.cs [hunk 2]
```diff
@@ -280,5 +293,16 @@ public static bool IsGuid(string value)
...
+
+            if (isAdmin) return true;
+
+            var domain = email.Split('@').Last();
+            return domain == LykkeEmailDomains.LykkeCom || domain == LykkeEmailDomains.LykkexCom;
```
Comment (justinlevinson) [inline src/CompetitionPlatform/Controllers/UserProfileController.cs]: I'd break this up, this syntax is confusing.
```
if(domain == LykkeEmailDomains.LykkeCom || domain == LykkeEmailDomains.LykkexCom)
  return true;
return false;
```

File: src/CompetitionPlatform/Controllers/UserProfileController.cs [hunk 3]
```diff
@@ -280,5 +293,16 @@ public static bool IsGuid(string value)
             Guid x;
             return Guid.TryParse(value, out x);
         }
+
+        private async Task<bool> UserIsLykkeMember(string email)
```
Comment (justinlevinson) [inline src/CompetitionPlatform/Controllers/UserProfileController.cs]: If this is reworded to IsUserLykkeMember(string email) it becomes a question - I recommend this ))

File: src/CompetitionPlatform/Helpers/StreamsConstants.cs [hunk 4]
```diff
@@ -4,4 +4,23 @@ public static class StreamsRoles
     {
         public const string Admin = "ADMIN";
     }
+
+    public static class ResultVoteTypes
```
Comment (justinlevinson) [inline src/CompetitionPlatform/Helpers/StreamsConstants.cs]: Ah, I see. If we can't get enums to work, this is a close second. Gets us out of hardcoding 👍

File: src/CompetitionPlatform/wwwroot/css/addstyle.css [hunk 5]
```diff
@@ -61,3 +61,7 @@
 .project-extended-tag {
     margin-right: -8px;
 }
+
+.lykke-team-label {
+    margin-left: 0 !important;
```
Comment (justinlevinson) [inline src/CompetitionPlatform/wwwroot/css/addstyle.css]: I suppose we have to do a CSS review at some point. !important should send up a flag that something's wrong.

File: src/CompetitionPlatform/Controllers/UserProfileController.cs (hunk 1 above)
Comment (Tengiz-Merabishvili) [reply src/CompetitionPlatform/Controllers/UserProfileController.cs]: Yes, there are several admins with a different address.

**Pull request merged**